<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecSettingsWindow</class>
 <widget class="QWidget" name="RecSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="CamSpinBox" name="spinHRes">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
  </widget>
  <widget class="CamSpinBox" name="spinVRes">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOK">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame {
    background-color: rgb(85, 85, 85);
    background-position: center;
    background-image: url(frame-preview.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frameImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { background-color: rgba(255, 255, 255, 63); }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="CamSpinBox" name="spinHOffset">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>134</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="CamSpinBox" name="spinVOffset">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>134</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblX">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblX_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>144</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resolution</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOffset">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>105</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkCenter">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>134</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 40px; height: 40px; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CamLineEdit" name="lineRate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>295</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>999.990 000 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>265</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frame Rate</string>
   </property>
  </widget>
  <widget class="CamLineEdit" name="linePeriod">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>345</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4 000.000 01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdMax">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPS">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSec">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>350</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sec</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblExp">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="CamLineEdit" name="lineExp">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>990.00u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdExpMax">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>425</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSec_2">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>435</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sec</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboGain">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblGain">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analog Gain</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupCommonRes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Common Resolutions</string>
   </property>
   <widget class="QComboBox" name="comboRes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdRecMode">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Record
Modes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdDelaySettings">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trigger
Delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCenter">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>105</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDigGain">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblGain_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Digital Gain</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CamSpinBox</class>
   <extends>QSpinBox</extends>
   <header>camspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>CamLineEdit</class>
   <extends>QLineEdit</extends>
   <header>camLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdCancel</tabstop>
  <tabstop>cmdOK</tabstop>
  <tabstop>cmdRecMode</tabstop>
  <tabstop>cmdDelaySettings</tabstop>
  <tabstop>comboGain</tabstop>
  <tabstop>lineExp</tabstop>
  <tabstop>cmdExpMax</tabstop>
  <tabstop>linePeriod</tabstop>
  <tabstop>lineRate</tabstop>
  <tabstop>cmdMax</tabstop>
  <tabstop>chkCenter</tabstop>
  <tabstop>spinVOffset</tabstop>
  <tabstop>spinHOffset</tabstop>
  <tabstop>spinVRes</tabstop>
  <tabstop>spinHRes</tabstop>
  <tabstop>comboRes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
